spec/system/claims/support/schools/mentors/support_user_adds_a_mentor_spec.rb

```diff
@@ -272,13 +272,13 @@ def then_i_see_errors(errors)
   end
 
   def then_i_see_form_with_trn(trn)
-    expect(page.find("#claims-mentor-form-trn-field").value).to eq(trn)
+    expect(page.find("#claims-add-mentor-wizard-mentor-step-trn-field").value).to eq(trn)
```

Comment:
It would be nicer to use this:

https://github.com/DFE-Digital/itt-mentor-services/blob/2d7fdfc9b407cdebfe4e5fe8df62b7f8c2b09964/spec/system/claims/schools/mentors/add_a_mentor_spec.rb#L294-L296